alerts/alerts.yml

```diff
@@ -0,0 +1,53 @@
+groups:
+  - name: Distribution bot. Unexpected exceptions.
+    rules:
+      - alert: DistributionBotUnexpectedErrors
+        expr: increase(distribution_bot_unexpected_exceptions{service="nor-reward-distribution-bot"}[1h]) > 0
+        labels:
+          severity: critical
```

Comment:
Let's make it `medium`

Reply:
Changed to `major`, we don't use `medium` severity